__tests__/04.cross-contract-calls-with-fungible-token.ava.ts

```diff
@@ -153,7 +153,7 @@ test('Transfer call with burned amount', async t => {
     await registerUser(ft, defi);
 
     const result = await root
-      .createTransaction(ft)
+      .batch(ft)
```

Comment:
I'm not sure about this one, since it's returning a Transaction object.

Reply:
Oh this rename is here. I forgot to note in PR description:
![image](https://user-images.githubusercontent.com/13259400/164581202-0a5757d3-c9dc-4a31-9cb3-378b574df133.png)